materialize-stepper.js

```diff
@@ -249,6 +261,20 @@ $.fn.activateStepper = function(options) {
                return window[feedback].call();
             }
             form.submit();
+         } else {
+          inputs = $stepper.find(':input');
+          first_found = true
+          for (index = 0; index < inputs.length; ++index) {
+            if (!inputs[index].validity.valid) {
+              step = inputs[index].closest('.step')
+              if (first_found && settings.openStepOnValidationError) {
+                step_number = $stepper.children('.step').index($(step))+1;
+                $stepper.openStep(step_number)
+                first_found = false
+              }
+              $(step).removeClass('done').addClass('wrong');
+            }
```

Comment:
Maybe it would make sense to build a function (with the name of what this does, maybe `openFirstFoundInvalidStep`) and then bind this on the error event.